parity-util-mem/Cargo.toml

```diff
@@ -13,13 +13,8 @@ dlmalloc = { version = "0.1.3", features = ["global"], optional = true }
 wee_alloc = { version = "0.4.5", optional = true }
 lru = { version = "0.4", optional = true }
 hashbrown = { version = "0.6", optional = true }
-# from https://github.com/microsoft/mimalloc:
-# mimalloc can be built in secure mode,
-# adding guard pages, randomized allocation, encrypted free lists, etc.
-# to protect against various heap vulnerabilities.
-# The performance penalty is only around 3% on average over our benchmarks.
-mimallocator = { version = "0.1.3", features = ["secure"], optional = true }
-mimalloc-sys = { version = "0.1.6", optional = true }
+mimalloc = { version = "0.1.18", default-features = false, optional = true }
```

Comment:
secure feature is disabled as it adds a non-trivial overhead, see https://github.com/purpleprotocol/mimalloc_rust/pull/21